### PR TITLE
Redesigning Profile page (New UI)

### PR DESCRIPTION
**Pull request by aayush-05**
Redesigning the Profile page according to the New UI
image
image

### CODE REVIEW

**Comment by aayush-05** [comment]
@tachyons can I skip those space related errors by hound ? :P
Let me know if any changes required.

**Comment by tachyons** [comment]
No, don't skip them

**Comment by aayush-05** [comment]
Done @tachyons

File: app/views/users/logix/profile.html.erb [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
+        </div>
+        <div  id="profile-details" class="col-xs col-sm-12 col-md col-lg-7">
+          <h3><%= @profile.name %></h3> <hr>
+          <h4>Member Since</h4>
+          <p><%= @profile.member_since %></p>
+          <h4>Educational Institution</h4>
+          <p><%= @profile.educational_institute %></p>
+          <h4>Country</h4>
+          <p><%= @profile.country_name %></p>
+          <h4>Subscribed to mails ?</h4>
```
Comment (JoshHeng) [inline app/views/users/logix/profile.html.erb]: No question mark needed and capitalisation - `Subscribed To Emails?`

File: app/views/users/logix/profile.html.erb [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
+      </div>
       <div class="row">
-        <div class="col-lg-8">
-          <p><strong>Name: </strong><%= @profile.name %></p>
-          <p><strong>Member since: </strong><%= @profile.member_since %> days</p>
-          <p><strong>Educational Institution: </strong><%= @profile.educational_institute %></p>
-          <p><strong>Country: </strong><%= @profile.country_name %></p>
-          <p><strong>Subscribed to mails: </strong><%= @profile.mail_subscription %></p>
+        <div class="col-xs col-sm-12 col-md col-lg-4">
+        <%= image_tag @profile.profile_picture.url(:medium), style: "border: 0px; border-radius: 50%; margin: 30px;" %>
```
Comment (JoshHeng) [inline app/views/users/logix/profile.html.erb]: Might need to add an alt description for accessibility

File: app/views/users/logix/profile.html.erb (hunk 1 above)
Comment (aayush-05) [reply app/views/users/logix/profile.html.erb]: @nitin10s whatsay ?

**Comment by nitin10s** [comment]
Putting the name under the picture centre-aligned would be better. Also, I think putting all the elements inside a box would be better. Hey! I think I'll just make a wireframe quickly to finalize the design. It's no point trying everything in dev.

I'll send the wireframe and we'll see how things would come out and then you could modify accordingly.

**Comment by aayush-05** [comment]
@nitin10s is the wireframe ready for this ?

**Comment by nitin10s** [comment]
Been a little busy. Will share it in a day or two.

**Comment by aayush-05** [comment]
@tachyons  @nitin10s

**Comment by aayush-05** [comment]
The required changes are done @nitin10s . Please check
image

**Comment by aayush-05** [comment]
@tachyons fixed merge conflicts. Can be merged now

**Comment by aayush-05** [comment]
Merge conflicts fixed, ready to be merged @tachyons

**Pull request merged**